### PR TITLE
Conv is reimplemented, MaxPool's potencial bug is fixed

### PR DESCRIPTION
**Pull request by semihlab**
* it is tested on onnx 1.11.0
* 6.7x performance enhancements in onnx test cases

### CODE REVIEW

File: src/opset/Conv_11.c [hunk 1]
```diff
@@ -22,133 +22,44 @@
 #include <connx/connx.h>
 
 /*{% for DTYPE, TYPE in loop_types(FLOAT32, FLOAT64) %}*/
+#define MAX(a, b) ((a) > (b) ? (a) : (b))
```
Comment (tribela) [inline src/opset/Conv_11.c:25]: This is duplicated for all of `DTYPE` loop.
Move it above the template for-loop line

File: src/opset/Conv_11.c [hunk 2]
```diff
@@ -22,133 +22,44 @@
...
+#define MIN(a, b) ((a) < (b) ? (a) : (b))
+
+#include <stdio.h>
 // clang-format off
-static void _conv_{{ DTYPE | to_name }}({{TYPE}}* Y_flatten, {{TYPE}}* X_flatten, int32_t feature_dim,
-                                int32_t* feature_shape, connx_Iterator* x_iter, {{TYPE}}* W_flatten,
-                                int32_t* kernel_shape, int32_t* dilations) {
+static void conv_1d_{{DTYPE}}({{TYPE}}* Y_flatten, int32_t* Y_shape, {{TYPE}}* X_flatten, int32_t* X_shape, {{TYPE}}* W_flatten, int32_t* W_shape, int32_t* pads, int32_t* dilations, int32_t* strides) {
```
Comment (tribela) [inline src/opset/Conv_11.c:30]: Although We used `clang-format off` for template, Follow clang-format's linebreak is look better.
1. Remove `//clang-format off` line
2. Run clang-format
3. Fix malformed template and re-attach `//clang-format off` line

File: src/opset/Conv_11.c [hunk 3]
```diff
@@ -259,82 +170,78 @@ int Conv_{{op_version}}(connx_Graph* graph, __attribute__((unused)) uint32_t out
...
             }
-        }
 
-        // clang-format off
-        connx_Thread_run_all(run_{{DTYPE | to_name}}, work_count, works,
-                             sizeof(struct Parameter_{{DTYPE | to_name}}));
-        // clang-format on
+            break;
+        default:
+            connx_error("Conv: dimension is not supported yet: %d\n", feature_dim);
```
Comment (tribela) [inline src/opset/Conv_11.c:242]: For compatibility, leave original implementation and run at this point would be better. Especially, We don't have 2d+ on this PR

**Review by tribela — CHANGES_REQUESTED** [review]
Looks good, But Some changes are required

File: src/opset/Conv_11.c (hunk 1 above)
Comment (semihlab) [reply src/opset/Conv_11.c:26]: Done

File: src/opset/Conv_11.c (hunk 2 above)
Comment (semihlab) [reply src/opset/Conv_11.c]: Done!

File: src/opset/Conv_11.c (hunk 3 above)
Comment (semihlab) [reply src/opset/Conv_11.c:417]: 2d and 3d is reimplemented!

**Review by tribela — APPROVED** [review]

**Pull request merged**